src/components/ImageGallery.jsx

```diff
@@ -763,13 +763,14 @@ class ImageGallery extends React.Component {
   }
 
   canSlideLeft() {
-    const { infinite } = this.props;
-    return infinite || this.canSlidePrevious();
+    const { infinite, isRTL } = this.props;
+    //so basically is the list is right to left (isRTL), the logic is inversed
```

Comment:
grammar fix here:

`// When isRTL is true, the logic is reversed`

Reply:
Personally, I would rather have the comment removed because the code you've provided is self explaining. But, since there is a comment which will alert a contributor to read it, it should be clear and concise.

Reply:
alright a side note, it's my first time contributing to kindly excuse any inconvenience caused.

Reply:
No inconvenience at all, this is part of the code review process. Thank you for contributing!